saleor/product/models/base.py

```diff
@@ -226,19 +301,36 @@ class Meta:
         verbose_name_plural = pgettext_lazy('Product variant model', 'product variants')
 
     def __str__(self):
+        # type: () -> str
         return self.name or self.display_variant()
 
     def check_quantity(self, quantity):
+        # type: (int) -> None
+        """ Raise InsufficientStock if quantity is bigger that available
+
+        Arguments:
+        quantity -- quantity of Product Variant
```

Comment:
Do we need to have _a list_ of arguments for functions that only take one or two? I believe explaining that quantity is in fact quantity does not improve this docstring much.